scripts/coverage

```diff
@@ -205,55 +200,107 @@ class LLVM:
         self._cov(subcommand='show', *extras, **kwargs)
 
 
+# TODO: implement
+@dataclass
+class ProfDir:
+    cwd: Path
+    llvm: LLVM
+
+    def __post_init__(self) -> None:
+        self.cwd.mkdir(parents=True, exist_ok=True)
+
+    @property
```

Comment:
if this field accessed several times it might be be good to utilize cached_property

Reply:
No, I don't think that's a good idea. Sure, it's somewhat costly, but if used carefully, it won't do any harm at all. Just think of the subtle bugs that could happen with such a change.

Reply:
You mean that file list can change on the fly? I think if this is the case many things will get broken anyway. Another option to better describe the intention might be to use get_files or similar name instead. For me `property` always means  something cheap to recreate every time it is accessed. It is not that important though, I suppose there wouldn't be too much files anyway